src/app/components/alertifier.js

```diff
@@ -11,7 +11,7 @@ export const SALUTATION = 'Hey!';
  *
  * Usage: const myAlertifier = Alertifier(document.querySelector('#myAlertifier'));
  */
-export default Alertifier = (container) => {
+export default (container) => {
```

Comment:
Artem told me the same thing. It’s strange. In the end he separeted the declaration from export. His version is already in the master (I think that’s causing the collision with your branch).